### PR TITLE
test: fix unit tests

### PR DESCRIPTION
**Pull request by MarcL**
### Summary

The unit tests for this plugin have been failing for a while. With digging, it appears that the cleanup process of deleting the temporary directory takes more than 5 seconds which causes a timeout in the `afterEach` hook. In profiling the cleanup function, it appears to take ~5500 ms.

This PR increases adds an increased test timeout of 6 seconds to the `afterEach` function to ensure cleanup completes.

### Test plan

Ensure that the unit tests execute without timeout errors.

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal

image

### CODE REVIEW

**Review by sarahetter — DISMISSED** [review]

File: plugin/test/unit/helpers/config.spec.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import {
 } from '../../../src/helpers/config'
 
 const SAMPLE_PROJECT_DIR = `${__dirname}/../../../../demo`
-const TEST_TIMEOUT = 60_000
+const TEST_TIMEOUT = 80_000
```
Comment (ericapisani) [inline plugin/test/unit/helpers/config.spec.ts:15]: [sand] I don't think this increase is needed at this time now that the increased timeout is being applied to the `afterEach`. Is the motivation to increase this value just to make sure we don't run into this upper limit anytime soon?

**Review by ericapisani — APPROVED** [review]
One non-blocking question, but otherwise LGTM 🚀

File: plugin/test/unit/helpers/config.spec.ts (hunk 1 above)
Comment (MarcL) [reply plugin/test/unit/helpers/config.spec.ts]: Sounds sensible. I was being a bit too cautious for future us. I've reverted. 👍🏻

**Review by orinokai — DISMISSED** [review]
:shipit:

**Review by ericapisani — DISMISSED** [review]

**Review by ericapisani — APPROVED** [review]

**Pull request merged**